locales/it/index.yml

```diff
@@ -1,6 +1,8 @@
 ---
 global:
   accessibility:
+    expanded: Espanso
```

Comment:
```suggestion
    expanded: Paragrafo espanso
```
Could it be clearer we have a content that could be closed/opened?
Perhaps `Sezione` could be better?

Reply:
Maybe Paragraph better explains it has an expanded text below, I'll apply this changes in the next commit

Reply:
@Undermaken updated in 3f1674b